cli/tests/e2e/snapshots/test_output/test_yaml_metavariables/report.json

```diff
@@ -0,0 +1,308 @@
+{
+  "errors": [],
+  "paths": {
+    "_comment": "<add --verbose for a list of skipped paths>",
+    "scanned": [
+      "targets/yaml/target.yaml"
+    ]
+  },
+  "results": [
+    {
+      "check_id": "rules.yaml-key",
+      "end": {
+        "col": 13,
+        "line": 1,
+        "offset": 12
+      },
+      "extra": {
+        "fingerprint": "0x42",
+        "is_ignored": false,
+        "lines": "- key: \"one\"",
+        "message": "\"one\"\n",
+        "metadata": {},
+        "metavars": {
+          "$VALUE": {
+            "abstract_content": "\"one\"",
+            "end": {
+              "col": 13,
+              "line": 1,
+              "offset": 12
+            },
+            "start": {
+              "col": 8,
+              "line": 1,
+              "offset": 7
+            }
+          }
+        },
+        "severity": "ERROR"
+      },
+      "path": "targets/yaml/target.yaml",
+      "start": {
+        "col": 3,
+        "line": 1,
+        "offset": 2
+      }
+    },
+    {
+      "check_id": "rules.yaml-key",
+      "end": {
+        "col": 13,
+        "line": 2,
+        "offset": 25
+      },
+      "extra": {
+        "fingerprint": "0x42",
+        "is_ignored": false,
+        "lines": "- key: 'two'",
+        "message": "'two'\n",
+        "metadata": {},
+        "metavars": {
+          "$VALUE": {
+            "abstract_content": "'two'",
+            "end": {
+              "col": 13,
+              "line": 2,
+              "offset": 25
+            },
+            "start": {
+              "col": 8,
+              "line": 2,
+              "offset": 20
+            }
+          }
+        },
+        "severity": "ERROR"
+      },
+      "path": "targets/yaml/target.yaml",
+      "start": {
+        "col": 3,
+        "line": 2,
+        "offset": 15
+      }
+    },
+    {
+      "check_id": "rules.yaml-key",
+      "end": {
+        "col": 11,
+        "line": 4,
+        "offset": 45
+      },
+      "extra": {
+        "fingerprint": "0x42",
+        "is_ignored": false,
+        "lines": "- key: |\n    three",
+        "message": "|\n    three\n\n",
```

Comment:
In theory we should semantically transform those as just "three"

Reply:
Right now, our infrastructure is incompatible with having the contents of the metavariable differ in length from the range of the metavariable itself. As much as I would like to "simplify" the abstract content (and lift it from being the literal text of the range), it seems this would require larger changes in the core and the CLI.